.github/workflows/artillery-report-users-user_id-tasks-task_id.yaml

```diff
@@ -0,0 +1,71 @@
+name: Artillery Action to test endpoint function users-userId-tasks-taskId
+
+on:
+  push:
+    branches:
+      - development # On merge with master, call this action
+
+env:
+  NODE_VERSION: '14'  # The version of node we are using
+  NPM_CONFIG_PREFIX: "~/.npm-global" # Required to set path for artillery call
+  PYTHON_VERSION: '3.8' # The version of python we are using
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest # Tells GH actions what we'll be running everything on
+
+    steps:
+    # Checks out the repository
+    - name: 'Checkout Repository GitHub Action' 
+      uses: actions/checkout@v2
+
+    # This is the script that sets up node
+    - name: Setup Node ${{ env.NODE_VERSION }} Environment 
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+        check-latest: true
+    
+    - name: 'Run npm init'
+      run: npm init -y
+
+    - name: Run npm install artillery  
+      run: npm install -g artillery
+
+    # - name: Check global npm location
+    #   run: npm list -g
+
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d-%H-%M-%S')"
+
+    - name: Sleep
+      run: sleep 10s
+
+    # Call artillery
+    - name: Run Artillery Test
+      run: /home/runner/.npm-global/lib/node_modules/artillery/bin/artillery run --output artillery_${{ steps.date.outputs.date }}.json ./test/artillery/artillery-users-user_id-tasks-task_id.yaml
```

Comment:
Instead of /home/runner you could just use ~ which isn't a big deal, just wanted to point it out.

Reply:
I wasn't aware of that syntax, thanks Kevin I will implement that!